chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -159,66 +159,97 @@ const std::map<int, const std::vector<BravePrepopulatedEngineID>*>
 
 // A versioned map tracking the singular default search engine per-country.
 BravePrepopulatedEngineID GetDefaultSearchEngine(int country_id, int version) {
-  const BravePrepopulatedEngineID default_v6 =
```

Comment:
cpplint complained about a chunk of this, so I just ran the whole function through clang-format